images/modules/com.redhat.osl.cli.packager/module.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Apache Software Foundation (ASF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "com.redhat.osl.cli.packager"
+version: "1.33.0-snapshot"
+description: "Download kn-workflow cli binaries from a given repository"
+
+packages:
+  manager: microdnf
+  install:
+    - zip
+    - tar
+    - gzip
+    - wget
+envs:
+  - name: KN_WORK_DIR
+    value: "/opt/app-root/src/go/src/github.com/knative/client"
+    description: "Immutable working directory for this image, please don't change it"
+args:
+  - name: KN_ARTIFACT_URL
+    description: "Base URL from where to download the artifacts"
+    value: "http://download.eng.bos.redhat.com/staging-cds/etera/openshift-serverless-clients/1/1.11/1.11.2-4/signed"
+    example: "http://download.eng.bos.redhat.com/staging-cds/etera/openshift-serverless-clients/1/1.11/1.11.2-4/signed"
```

Comment:
Not sure about the current policy, if we can use internal URLs in public repos?

Reply:
these links or refs should be externalized through properties.

Reply:
These links are externalized, this is just a default value. There's a `build-args` argument as I highlighted in the PR description.